realm/realm-library/src/main/cpp/io_realm_internal_core_NativeMixed.cpp

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "io_realm_internal_core_NativeMixed.h"
+
+#include "java_accessor.hpp"
+
+using namespace std;
+using namespace realm;
+using namespace realm::_impl;
+
+static void finalize_mixed(jlong ptr) {
+    delete reinterpret_cast<Mixed *>(ptr);
+}
+
+JNIEXPORT jlong JNICALL
+Java_io_realm_internal_core_NativeMixed_nativeGetFinalizerPtr(JNIEnv *, jclass) {
+    return reinterpret_cast<jlong>(&finalize_mixed);
+}
+
+JNIEXPORT jlong JNICALL
+Java_io_realm_internal_core_NativeMixed_nativeCreateMixedNull(JNIEnv *, jclass) {
+    return reinterpret_cast<jlong>(new Mixed());
```

Comment:
All of these methods needs to be wrapped in:

```
try {

}
CATCH_STD()
```
Otherwise, we risk letting native crashes hit the user